pact/Pact/Core/IR/Eval/CEK.hs

```diff
@@ -2022,10 +2026,15 @@ isKeysetInSigs info cont handler env (KeySet kskeys ksPred) = do
   where
   matchKey k _ = k `elem` kskeys
   atLeast t m = m >= t
+  elide pk | T.length pk < 8 = pk
+           | otherwise = T.take 8 pk <> "..."
```

Comment:
is the first case ever possible?

Reply:
I was wondering about this too. Its a direct port of legacy pact.